tests/test_loongarch.c

```diff
@@ -0,0 +1,145 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <capstone/capstone.h>
+#include <capstone/platform.h>
+
+struct platform {
+  cs_arch arch;
+  cs_mode mode;
+  unsigned char *code;
+  size_t size;
+  const char *comment;
+};
+
+static csh handle;
+
+static void print_string_hex(const char *comment, unsigned char *str,
+                             size_t len) {
```

Comment:
please have the closing bracket `{` on the new line.

Reply:
@aquynh This is not consistent through other modules. Also, not in old ones like `x86`. So I would suggest making this style change when we format all code with `clang-format` before the `v6` release.

Reply:
ok this can be done with clang-format, but we want the PRs to be mindful about the coding convention.